shift_detector/precalculations/Store.py

```diff
@@ -36,3 +48,12 @@ def __getitem__(self, needed_preprocessing) -> DataFrame:
         preprocessing = needed_preprocessing.process(self)
         self.preprocessings[needed_preprocessing] = preprocessing
         return preprocessing
+
+    @staticmethod
+    def verify_min_data_size(size):
+        if size < MIN_DATA_SIZE:
+            raise InsufficientDataError('The input data is insufficient for the column type heuristics to work. Only '
+                                        '{actual} row(s) were passed. Please pass at least {expected} rows.'.format(
+                                            actual=size,
+                                            expected=MIN_DATA_SIZE),
```

Comment:
indent seems to be a bit broken here

Reply:
This indent was intended to separate between the params of the error and the format() method. Open to better suggestions though

Reply:
ah okay now I get it.